hack/test-end-to-end.sh

```diff
@@ -358,6 +358,13 @@ echo "[INFO] Validating exec"
 registry_pod=$(osc get pod | grep deployment=docker-registry | grep docker-registry | awk '{print $1}')
 osc exec -p ${registry_pod} whoami | grep root
 
+# Diagnostics
+echo "[INFO] Running openshift-diagnostics"
+# show only errors, fail if any errors
+wait_for_command "openshift-diagnostics --config=${OPENSHIFTCONFIG} --master-config=${MASTER_CONFIG_DIR}/master-config.yaml --node-config=${NODE_CONFIG_DIR}/node-config.yaml --diaglevel=0" 60
```

Comment:
I'll revisit client config loading. It may already work or there is likely something reasonable to do here.